src/zenml/integrations/kubernetes/flavors/kubernetes_orchestrator_flavor.py

```diff
@@ -49,19 +51,54 @@ class KubernetesOrchestratorConfig(  # type: ignore[misc] # https://github.com/p
     """Configuration for the Kubernetes orchestrator.
 
     Attributes:
-        kubernetes_context: Optional name of a Kubernetes context to run
-            pipelines in. If not set, the current active context will be used.
-            You can find the active context by running `kubectl config
-            current-context`.
+        kubernetes_context: Name of a Kubernetes context to run pipelines in.
         kubernetes_namespace: Name of the Kubernetes namespace to be used.
             If not provided, `zenml` namespace will be used.
-        skip_config_loading: If `True`, don't load the Kubernetes context and
-            clients. This is only useful for unit testing.
+        local: If `True`, the orchestrator will assume it is connected to a
+            local kubernetes cluster and will perform additional validations and
+            operations to allow using the orchestrator in combination with other
+            local stack components that store data in the local filesystem
+            (i.e. it will mount the local stores directory into the pipeline
+            containers).
+        skip_local_validations: If `True`, the local validations will be
+            skipped.
     """
 
-    kubernetes_context: Optional[str] = None  # TODO: Potential setting
+    kubernetes_context: str  # TODO: Potential setting
```

Comment:
What is the comment for?

Reply:
This was here already related to an old PR where the configuration of some components is now an instance of `BaseSettings` which can be set on a pipeline level. This can be beneficial in the context where running different pipelines in different clusters (Dev, QA, Prod)